index.html

```diff
@@ -156,6 +156,20 @@ <h1 class="mt-3 mb-3">Random Image Picker</h1>
                         <img id="random-image" class="img-thumbnail random-image" src="" alt="Randomly picked image.">
                     </div>
                 </div>
+                <div class="col-lg-4 mb-4 text-center">
+                    <div class="row ml-5">
+                        <h2 class="mt-3 mb-4">History:</h2>
+                        <div class="col-12 mb-4 history-image-wrapper">
+                            <img id="previous-image-1" class="img-thumbnail history-image" src="" alt="Previously picked image 1" style="display: none;">
+                        </div>
+                        <div class="col-12 mb-4 history-image-wrapper">
+                            <img id="previous-image-2" class="img-thumbnail history-image" src="" alt="Previously picked image 2" style="display: none;">
+                        </div>
+                        <div class="col-12 mb-4 history-image-wrapper">
+                            <img id="previous-image-3" class="img-thumbnail history-image" src="" alt="Previously picked image 3" style="display: none;">
+                        </div>
+                    </div>
+                </div>
```

Comment:
I would propose to have all extra features in a feature toggle on the home screen. E.g. `show directly`.

Reply:
Another option would be to allow for the default view (everyone is used to), and hide the history by default. Allowing people to show the history by, e.g. a window in the right side or so.